src/components/notifications/Whapi.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="mb-3">
+        <label for="whapi-api-url" class="form-label">{{ $t("API URL") }}</label>
+        <input id="whapi-api-url" v-model="$parent.notification.whapiApiUrl" type="text" class="form-control" required>
+    </div>
+
+    <div class="mb-3">
+        <label for="whapi-auth-token" class="form-label">{{ $t("Token") }}</label>
+        <input id="whapi-auth-token" v-model="$parent.notification.whapiAuthToken" type="text" class="form-control" required>
+    </div>
+
+    <div class="mb-3">
+        <label for="whapi-to-number" class="form-label">{{ $t("Phone number") }}</label>
+        <input id="whapi-to-number" v-model="$parent.notification.whapiToNumber" type="text" pattern="^[\d-]{10,31}$" class="form-control" required>
+        <i18n-t tag="div" keypath="wayToWriteWhapiPhoneNumber" class="form-text"></i18n-t>
```

Comment:
Please component translations only if you need to embed html in this translation.

```suggestion
        <div class="form-text">{{ t("wayToWriteWhapiPhoneNumber") }}</div>
```

Please also consider if adding an example phone number might make setting this up eazier like this and including it via `{0}` in the translation

```suggestion
        <div class="form-text">{{ t("wayToWriteWhapiPhoneNumber", ["00117612345678"]) }}</div>
```